content/en/docs/concepts/sampling/index.md

```diff
@@ -45,6 +32,70 @@ Sometimes, the definitions of these terms get mixed up. You might find someone
+without losing visbility. Although there are other ways to lower costs, such as
+filtering or aggregating data, these other methods do not adhere to the concept
+of _representativeness_, which is crucial when performing in-depth analysis of
+application or system behavior.
+
+Representativeness is the principle that a smaller group can accurately
+represent a larger group. Additionally, representativeness can be mathematically
+verified, meaning that you can have high confidence that a smaller sample of
+data accurately represents the larger group.
+
+Additionally, the more data you generate, the less data you actually need to
+have a representative sample. For high-volume systems, is quite common for a
+sampling rate of 1% or lower to very accurately represent the other 99% of data.
+
+### When to Sample
+
+You should consider sampling if you meet any of the following criteria:
+
+- You generate 1000 or more traces per second
+- Most of your trace data represents healthy traffic with little variation in
+  data
+- You have some common criteria, like errors or high latency, that usually means
+  something is wrong
+- You have domain-specific criteria you can use to determine relevant data
+  beyond errors and latency
+- You can describe some common rules that determine if data should be sampled or
+  dropped
+- You have a tell your services apart, so that high- and low-volume services are
+  sampled differently
+- You have the ability to route unsampled data (for "just in case" scenarios) to
+  low-cost storage systems
+
+Finally, consider your overall budget. If you have limited budget for
+Observability, but can afford to spend time to effectively sample, then sampling
+can generally be worth it.
+
+### When not to Sample
+
+Sampling may not be appropriate for you. You may want to avoid sampling if you
+meet any of the following criteria:
+
+- You generate very little data (tens of small traces per second or lower)
+- You only use Obsevability data in aggregate, and can thus pre-aggregate data
+- You are bound by circumstances such as regulation that prohibit dropping data
+  (and cannot route unsampled data to low-cost storage)
+
+Finally, you should consider the following three costs associated with sampling:
+
+1. The direct cost of compute to effectively sample data, such as a tail
+   sampling proxy
+2. The indirect engineering cost of maintaining effective sampling methodologies
+   as more application, systems, and data are involved
+3. The indirect opportunity cost of missing critical information with
+   _ineffective_ sampling techniques
+
+Sampling, while effective and reducing Observability costs, can introduce other
+unexpected costs if not performed well. It may be cheaper to simply "pay for
+more Observability" (be it with a vendor or compute when self-hosting) depending
+on your Observability backend, the nature of your data, and your attempts to
+sample effectively.
```

Comment:
```suggestion
Sampling, while effective and reducing observability costs, might introduce other
unexpected costs if not performed well. It could be cheaper to allocate more
resources for observability instead, either with a vendor or compute when
self-hosting, depending on your observability backend, the nature of your data,
and your attempts to sample effectively.
```